content/_index.md

```diff
@@ -7,10 +7,28 @@ BookToC: false
 
 # The Linux Userspace API (UAPI) Group
 
-The userspace API ("uapi") group is a community for people with an interest in innovating how we build, deploy, and run modern Linux operating systems.
+The userspace API ("uapi") group is a community for people with an interest in innovating how we build, deploy, run, and safely update modern Linux operating systems.
 It serves as a central gathering place for specs, documentation, and ideas.
 
-Follow [@uapi-group](https://twitter.com/uapi_group) on twitter to catch the latest news!
+The group is in the process of constituting itself and is exploring the problem space relevant to us.
+In the general scope of the group are trusted boot mechanisms, kernel / disk image formats, TPM usage, safe and secure updates, and related topics.
+The group also cares about interop of image-based and/or immutable Linux distributions.
+The group is in its inception, so the list of topics is by no means complete - other relevant topics related to the above might be added in the future.
+
+The UAPI group aims to establish specifications for common concepts shared among relevant parties.
+We are open to anyone who has an interest in constructive collaboration to to join the discussions, and to represent the directions and interests of their projects and distributions.
+
+## Engage with the Group
+
+Our main means of engagement is Github.
+Please check out the group's [Github discussions](https://github.com/orgs/uapi-group/discussions) for general ongoing conversations - don't hesitate to tune in and share your thoughts!
+Feel free to start a new discussion if none exists for whatever brings you here - but please keep the group's focus in mind and do not stray off-topic.
+
+Check out the group's repos at the [UAPI group github org](https://github.com/uapi-group) to get an overview of existing work.
```

Comment:
We're using non-capital letters. IOW, `uapi`, never `UAPI`.

Reply:
Updated.